.changeset/soft-shrimps-beg.md

```diff
@@ -0,0 +1,13 @@
+---
+"@rocket.chat/meteor": patch
+"@rocket.chat/fuselage-ui-kit": patch
+"@rocket.chat/gazzodown": patch
+"@rocket.chat/ui-avatar": patch
+"@rocket.chat/ui-client": patch
+"@rocket.chat/ui-composer": patch
+"@rocket.chat/ui-video-conf": patch
+"@rocket.chat/uikit-playground": patch
+"@rocket.chat/ui-theming": patch
+---
+
+fix: omnichannel department names overflowing, adding customized options in some omnichannel selects. 
```

Comment:
Please, let's add a more explanatory changeset since this is intended for customers.
(reference [here](https://developer.rocket.chat/contribute-to-rocket.chat/modes-of-contribution/participate-in-rocket.chat-development/development-workflow#id-4.-adding-changeset-to-your-pull-request) )